agent/command/s3_put.go

```diff
@@ -87,6 +87,12 @@ type s3put struct {
 	// for missing files.
 	Optional string `mapstructure:"optional" plugin:"expand"`
 
+	// Patchable defaults to true. If set to false, this command will noop without error for patch tasks.
+	Patchable string `mapstructure:"patchable" plugin:"patchable"`
+
+	// PatchOnly defaults to false. If set to true, this command will noop without error for non-patch tasks.
+	PatchOnly string `mapstructure:"patch_only" plugin:"patch_only"`
```

Comment:
since patchable and patchOnly seem like inverses of each other, what is the benefit of having both?

Reply:
wait no, never mind, lol